src/commondao/common.dao.model.ts

```diff
@@ -77,6 +77,16 @@ export interface CommonDaoCfg<
 
   excludeFromIndexes?: (keyof DBM)[]
 
+  /**
+   * @default to false
+   * Set to true to limit DB writing:
+   * * Will throw an error if an object with matching ID already exists during save().
+   * * saveBatch, delete*() and patch() will throw.
```

Comment:
Why do you want saveBatch to throw? Asking, because I wasn't expecting this behavior. On the high level, if something is immutable, I'd expect all methods to respect/support it.

Reply:
For delete - I agree, ok to throw, since the Table is declared immutable.

Reply:
For patch... Current behavior of patch is that it "loads if object exists" and then patches it and saves. But it allows object to be not present (works like "upsert").

But semantically, I agree that patch implies "mutating" the object (for cases when it exists), so, yes, in the end I agree to throw there.

Reply:
UPD: ok, now I see that for saveBatch you throw the "unsupported error", this explains my question. Perfectly fine to not support features that are "not needed yet" (or, too complex to implement before we actually need it).